Core/NoOpRunner.Core/Shapes/PowerUp.cs

```diff
@@ -6,19 +6,23 @@ namespace NoOpRunner.Core.Shapes
 {
     public class PowerUp : BaseShape
     {
-        public PowerUp(int centerPosX, int centerPosY) : base(centerPosX, centerPosY)
-        {
-            Color powerup = GetRandomPowerUp();
+        public readonly PowerUps powerup;
```

Comment:
This variable should be named Type or PowerUpType

Reply:
Fixed